docs/MediaCodecTests.md

```diff
@@ -0,0 +1,90 @@
+# How to use MediaCodec Tests
+
+## Background
+
+In Android Media, there is a codec framework team and a video codec quality team. There was a gap in testing for audio codec quality. We wanted to create a tool to find and reproduce audio codec quality issues.
```

Comment:
Rather than discuss our gaps, I would start with a summary of what the test does, who is it for, and what benefit will they get from using it. Then they can quickly decide whether to read the rest of the doc.